src/app/public/plugin-resources/code-examples/input-box/basic/input-box-demo.component.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Component
+} from '@angular/core';
+
+import {
+  SkyTheme,
+  SkyThemeMode,
+  SkyThemeSettings
+} from '@skyux/theme';
+
+@Component({
+  selector: 'app-input-box-demo',
+  templateUrl: './input-box-demo.component.html'
+})
+export class InputBoxDemoComponent {
+
+  /**
+   * The Input box component works best in modern theme. This is purely for demonstration purposes.
```

Comment:
Done.

Reply:
In a recent discussion with @Blackbaud-AdamFunderburk, he mentioned that when talking about components, we should probably refer to them using title case, so it would be "Input Box" everywhere. I'm not sure if we want to start that trend here or if it warrants more discussion.

Reply:
I talked to Adam too about the idea of capitalizing component names as proper nouns and plan to bring it up with the team as a flextime discussion. I just didn't get a chance to do so this week. I looked into the Angular docs to see if they support this, but they do NOT capitalize their component names as proper nouns. So if we want to be consistent with Angular, we probably don't want to make the change. I usually prefer not designate things as proper nouns, but if the team wants to and we have a good rationale, I'm all for switching our style. That said, I think we should discuss this as a team first and make sure we have a clear style before we start switching things up. After that, Denise or I can clean up the docs to implement any new style rather than doing it one-off in PRs like this. That's my 2 cents at least.

Reply:
I'm a big advocate of this change and I believe it will help the readability of sentences with component names in them. I'll let @blackbaud-johnly take the lead on this decision, though.

Reply:
I'm going to resolve this comment thread, and we'll pursue this capitalization idea outside of this PR.